### DISCUSSION TITLE
Color of the weather in sidebar

### DISCUSSION THREAD

**Discussion by tunguskar** [matt8707/hass-config#126] [Q&A] [answered]
I searched a lot but haven't found the setting for the Color of the text of the weather in the sidebar. Can you give me a hint? Also the font size of the custom:button-card please. Besides great job really beautiful!

**Comment by matt8707** [comment, marked as answer]
> I searched a lot

did you now

https://github.com/matt8707/hass-config/blob/cb41a584dc3016765fdb2719dc920abd1d7263b1/button_card_templates/sidebar.yaml#L54

https://github.com/matt8707/hass-config/blob/cb41a584dc3016765fdb2719dc920abd1d7263b1/themes.yaml#L54

**Reply by tunguskar** [reply]
that did the trick thanks, but can you give me a link or something to read about

```yaml
    @supports not (-moz-appearance:none) {
      .time-colon {
        position: relative;
        top: -.11em;
      }
    }
    p {
      font-size: var(--sidebar-font-size);
      line-height: var(--sidebar-line-height);
      font-weight: 250;
      letter-spacing: 0.07vw;
      rgba(255, 255, 255, 0.8);
    }
    b {
      font-weight: 700;
    }
```

what does p and b mean?
